src/primitives/index.js

```diff
@@ -0,0 +1,36 @@
+import reactPrimitives from 'react-primitives'
+import glamorous, {ThemeProvider, withTheme} from '../index'
+
+// Create glamorous component with composition
+function createComponent(componentConstructor, tag) {
```

Comment:
> if we could fix the invalid 'tag' type.

Could you elaborate? I don't see any harm in just making the code simpler. No need to have the extra function...